scripts/zones/Dynamis-Bastok/mobs/Vanguard_Militant.lua

```diff
@@ -1,18 +1,9 @@
 -----------------------------------
 -- Area: Dynamis Bastok
---  MOB: Vanguard Militant
+--  MOB: Vanguard_Militant
```

Comment:
why adding the underscore? I don't mind it when we fix complete wrong ones by copying out of the sql but don't see a reason to put it in when the name was otherwise accurate. Find&replace regex happen here?

Reply:
Heh. I wish it were find/replace.  It was more like ... I got Dynamis Bastok and Dynamis Jeuno folders mixed up somehow (see above) and deleted all the Vanguard scripts from Bastok's folder.  So I copied them all over from another Dynamis folder and changed the Area to Dynamis Bastok all sneaky-like.